### PR TITLE
CO-16 Calculate fuel consumption

### PR DESCRIPTION
**Pull request by JeppeTrip**

### CODE REVIEW

**Comment by objmj-itminds** [comment]
@JeppeTrip, Can you please resolve merge conflicts in mock DB so that all tests pass?

File: backend/Application/Locations/Commands/AddLocationImageCommand.cs [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+        var filename = request.LocationId + "." + imgType;
+        string filePath = Path.Combine(_options.LocationPath, filename);
+
+        if (System.IO.File.Exists(filePath))
+        {
+          throw new ArgumentException("Image with " + request.LocationId + " already exists.");
+        }
+
+        using (Stream fileStream = new FileStream(filePath, FileMode.Create))
```
Comment (CSCITM) [inline backend/Application/Locations/Commands/AddLocationImageCommand.cs:59]: Can throw exception if there is missing write permissions etc.

File: backend/Domain/EntityExtensions/RegionExtensions.cs [hunk 2]
```diff
@@ -0,0 +1,24 @@
+using System;
+using System.Linq;
+using Domain.Entities;
+
+namespace Domain.EntityExtensions
+{
+  public static class RegionExtensions
+  {
+    private const int NUM_OBSERVATIONS = 5;
```
Comment (CSCITM) [inline backend/Domain/EntityExtensions/RegionExtensions.cs:9]: Could be moved to appsettings if this ever needs to be changed.

File: backend/Domain/EntityExtensions/LocationExtentions.cs [hunk 3]
```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Linq;
+using Domain.Entities;
+namespace Domain.EntityExtensions
+{
+  public static class LocationExtensions
+  {
+    public static double FuelConsumptionPerDegreeOfHeating(this Location location)
+    {
+      const int HEAT_BASE = 21;
```
Comment (CSCITM) [inline backend/Domain/EntityExtensions/LocationExtentions.cs:10]: Could be moved to appsettings if this ever needs to be changed.

**Review by CSCITM — COMMENTED** [review]
I have only reviewed the backend part.

**Review by objmj-itminds — APPROVED** [review]

**Pull request merged**